### DISCUSSION TITLE
Syncing issue for multiple nodes running on private IPs

### DISCUSSION THREAD

**Discussion by elamirch** [Chia-Network/chia-blockchain#4937] [Support]
I've read on the community that if there is more than 1 node farming on an IP they'll stop syncing as it's better to use one node for plotting and use the rest of the computers just to store plots, in Iran internet system is designed in a way that we have colonies of private IP addresses (usually 10.*.*.*) being connected to IP addresses provided by ISPs, so it's somehow a cluster of private IP islands, I don't know if this is the reason I can't sync to other nodes since 3 days ago or not but if yeah I'm just reporting :)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
